webpack.mix.js

```diff
@@ -1,11 +1,16 @@
 let mix = require('laravel-mix');
+let eslint = require('eslint-webpack-plugin');
 
 mix
-  .js('resources/js/app.js', 'resources/static/js')
-  .setPublicPath('resources/static')
-  .postCss('resources/css/app.css', 'resources/static/css', [
-    require('tailwindcss'),
-  ]);
+    .js('resources/js/app.js', 'resources/static/js')
+    .setPublicPath('resources/static')
+    .postCss('resources/css/app.css', 'resources/static/css', [
+        require('tailwindcss'),
+    ]);
 
-  mix.browserSync('127.0.0.1');
-  mix.disableSuccessNotifications();
+mix.browserSync('127.0.0.1');
+mix.disableSuccessNotifications();
```

Comment:
I think this code should be in `if` condition and use if app is not in production.

Reply:
done